.gitignore

```diff
@@ -0,0 +1,20 @@
+.idea
```

Comment:
is this `.gitignore` taken from some default/template? does not seem like we need many of the stuffs here.

Reply:
updated